wrappers/python/nntile/layer/batch_norm.py

```diff
@@ -0,0 +1,200 @@
+# @copyright (c) 2022-present Skolkovo Institute of Science and Technology
+#                              (Skoltech), Russia. All rights reserved.
+#                2023-present Artificial Intelligence Research Institute
+#                              (AIRI), Russia. All rights reserved.
+#
+# NNTile is software framework for fast training of big neural networks on
+# distributed-memory heterogeneous systems based on StarPU runtime system.
+#
+# @file wrappers/python/nntile/layer/layer_norm.py
+# LayerNorm of NNTile Python package
+#
+# @version 1.0.0
+
+import math
+
+from nntile.layer.base_layer import BaseLayer
+from nntile.tensor import (
+    Tensor,
+    TensorMoments,
+    add_async,
+    add_fiber_async,
+    hypot_scalar_inverse_async,
+    norm_slice_async,
+    ones,
+    pow_async,
+    prod_async,
+    prod_fiber_async,
+    sum_fiber_async,
+    zeros,
+)
+
+
+class BatchNorm2d(BaseLayer):
+    def __init__(
+        self,
+        x: TensorMoments,
+        weight: Tensor,
+        bias: Tensor,
+        x_res_tm: TensorMoments,
+        x_mean: Tensor,
+        inv_std: Tensor,
+        eps: float = 1e-05,
+        redux: bool = False,
+    ):
+        self.x_tm = x
+        self.x = x.value
+        self.eps = eps**0.5  # we will square it in forward (implementation constraints)
+
+        # TODO: add usage of learnable transforms
```

Comment:
You can take a look how it is done LayerNorm

Reply:
done